source/common/data.json

```diff
@@ -149,5 +149,20 @@
       "executivepaper": "Executive",
       "legalpaper": "Legal"
     },
-    "events": []
+    "events": [],
+    "highlightingModes": {
+      "javascript": [ "javascript", "js", "node", "json", "typescript", "ts"],
+      "clike": [ "c", "c\\+\\+", "cpp", "c\\#", "csharp", "java", "kotlin", "objective-c", "objectivec", "scala" ],
```

Comment:
special characters such a `+` or `#` must be escaped here.

Reply:
… because this way we can immediately stuff them into a regexp, yes! I think it's fine to have it there because having to compile an escaping regex to parse the trigger words in the mode loader would add nothing but computational overhead, I think it's clear enough that these are regex-escapers!